docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       POSTGRES_HOST_AUTH_METHOD: trust
 
   conjur:
-    image: cyberark/conjur:1.9
+    image: cyberark/conjur:1.11.4
```

Comment:
this should be a separate PR IMO

Reply:
we changes the tests so it woll work with appliance it will be too much hustle now

Reply:
dont understand how it is related. This is related to ubuntu integration tests, not rhel tests

Reply:
@eranha ?

Reply:
in conjur 1.9 the error was server error 500 and it changed in appliance because of different conjur version

Reply:
so I still think this and the tests you rightfully corrected should be in a separate PR. This is not a minor task in my opinion that should be coupled with this PR because it effects our test plan